tensorflow/docs_src/deploy/hadoop.md

```diff
@@ -52,10 +52,16 @@ be set:
     classpath wildcard manually. For more details, see
     [HADOOP-10903](https://issues.apache.org/jira/browse/HADOOP-10903).
 
-If the Hadoop cluster is in secure mode, the following environment variable must
-be set:
+If the Hadoop cluster is in secure mode, one the following environments variable
+should be set:
+*   **KRB5CCNAME**: The build in Kerberos environment variable for ticket cache file.
+For example:
 
-*   **KERB_TICKET_CACHE_PATH**: The path of Kerberos ticket cache file. For example:
+    ```shell
+    export KRB5CCNAME=/tmp/krb5cc_10002
+    ```
+*   **KERB_TICKET_CACHE_PATH**: The path of Kerberos ticket cache file, duplicate
```

Comment:
> Can we just delete the mention of the old environment variable?
> 
> Can you also add a TODO comment next to KERB_TICKET_CACHE_PATH in the code to mention that it should be deleted later?

OK, thanks for your reply, according to your advice,  I'll retry to update this commit.